riscv-smtdeleg-sstcfg.adoc

```diff
@@ -0,0 +1,98 @@
+[[intro]]
+== Introduction
+
+For a Machine-level environment, extension *Smtdeleg* (‘Sm’ for Privileged architecture and Machine-level extension, ‘tdeleg’ for Trigger Delegation) encompasses all added CSRs and all behavior modifications for a hart, over all privilege levels. For a
+Supervisor-level environment, extension *Sstcfg* (‘Ss’ for Privileged architecture and Supervisor-level extension, ‘tcfg’ for Trigger Configuration) provides access to delegated triggers. These extensions depend on the Sdtrig and Sscsrind extensions.
```

Comment:
Do we need to add dependency on Smstateen extension in addition to Sscsrind?

Reply:
Technically it doesn't depend on Smstateen, even though any practical implementation will want to include it.  One could add the extension without Smstateen, which would mean that triggers are always accessible from S-mode and VS-mode.